meson.build

```diff
@@ -79,8 +67,11 @@ if run_command('checkdir.py', 'libs/lczero-common/proto').returncode() != 0
   endif
 endif
 
-pb_files = gen.process('libs/lczero-common/proto/net.proto',
+pb_files = [
+  'src/utils/protomessage.cc',
+  gen.process('libs/lczero-common/proto/net.proto',
   preserve_path_from : meson.current_source_dir() + '/libs/lczero-common/')
```

Comment:
The `preserve_path_from` etc. is no longer needed.

Reply:
I think it is needed. Otherwise meson will expect it in the root of the built directory rather than in proto/ subdir.

Reply:
I wasn't sure myself, so I tested it before posting. Turn out we needed it only for the generated c++ file and not the header.